src/debugger/debugAdapterFactory.ts

```diff
@@ -12,6 +11,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Why not just remove the unused arguments from the `createDebugAdapterDescriptor` and `resolveDebugConfiguration` functions if they aren't being used?

Reply:
Good call, done.

Reply:
Is this line necessary anymore after removing them?